bindings/stronghold-nodejs/.cargo/config.toml

```diff
@@ -0,0 +1,7 @@
+[target.aarch64-unknown-linux-gnu]
```

Comment:
Our current target are 
´´´
"aarch64-apple-darwin",
"x86_64-apple-darwin",
"x86_64-pc-windows-msvc",
"x86_64-unknown-linux-musl",
"x86_64-unknown-linux-gnu"
´´´
so I think this can be removed